src/components/modules/chat/server/chat.firebase.ts

```
@@ -0,0 +1,143 @@
+  });
+  return chatDoc.id;
+};
+
+export const sendMessage = async (
+  chatId: string,
+  senderId: string,
+  text: string,
+  attachment?: { name: string; type: string; data: string },
+): Promise<void> => {
+  const messagesCol = collection(db, `chats/${chatId}/messages`);
+  const messageData: Record<string, any> = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)